app/upgrades/v16/upgrades.go

```diff
@@ -62,8 +61,8 @@ func CreateUpgradeHandler(
 		}
 
 		// Remove the deprecated governance proposals from store
-		logger.Debug("deleting deprecated proposals...")
-		DeleteDeprecatedProposals(ctx, gk, logger)
+		logger.Debug("deleting deprecated incentives module proposals...")
+		DeleteIncentivesProposals(ctx, gk, logger)
```

Comment:
ditto here

Reply:
same thing as above, not sure why we would prefer "wrong" v16 logic, that is not the actual logic in the handler 🤔